Unit 12. Generative models - Practical issues (#136)
johko/computer-vision-course: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx

# Privacy, Bias and Societal Concerns

The widespread adoption of AI-powered image editing tools raises significant concerns regarding privacy, bias, and potential societal ramifications. These tools, capable of manipulating both 2D and 3D images with remarkable realism, introduce ethical dilemmas and require careful consideration.

> 

Reviewer: ```suggestion
```

What you will learn from this chapter:

>     - Impact of such AI images/videos on Society

Reviewer: ```suggestion
- Impact of such AI images/videos on society
```
You can remove the indentations and capitals mid-sentence

    - Current Approaches to tackle the issues

>     - Future Scope

Reviewer 2: I would replace the capital letters also here ("Approaches" --> "approaches" and "Scope" --> "scope")

## Impact on Society
The ability to effortlessly edit and alter images has the potential to:

> - **Undermine trust in media**: Deepfakes, convincingly manipulated videos, can spread misinformation and erode public trust in news and online content.

Reviewer: ```suggestion
- **Undermine trust in media:** Deepfakes, convincingly manipulated videos, can spread misinformation and erode public trust in news and online content.
```

- **Harass and defame individuals**: Malicious actors can use AI tools to create fake images for harassment, defamation, and other harmful purposes.
- **Create unrealistic beauty standards**: AI tools can be used to edit images to conform to unrealistic beauty standards, negatively impacting self-esteem and body image.

## Current Approaches
Several approaches are currently being employed to address these concerns:

- **Transparency and labeling**: Platforms and developers are encouraged to be transparent about the use of AI-edited images and implement labeling systems to differentiate real and manipulated content.
- **Fact-checking and verification**: Media outlets and tech companies are investing in fact-checking and verification tools to help combat the spread of misinformation and disinformation.
- **Legal frameworks**: Governments are considering legislative measures to regulate the use of AI-edited images and hold individuals accountable for their misuse.

## Future Scope
The future of AI-edited images will likely involve:

> - **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develop sophisticated realistic AI images generating algorithms whereas another group develop methods to identify them

Reviewer 3: ```suggestion
- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develops sophisticated realistic AI images generating algorithms whereas another group develops methods to identify them
```

Reviewer 2: "But is like a cat and dog game" --> "But **it's** like a **cat-and-mouse** game"

> - **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develop sophisticated realistic AI images generating algorithms whereas another group develop methods to identify them

Reviewer: ```suggestion
- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But it is like a cat and dog game: where one group develops sophisticated, realistic image generation algorithms, whereas another group develops methods to identify them.
```

> - **Public awareness and education**: Public awareness campaigns and educational initiatives will be crucial in promoting responsible use of AI-edited images and combating the spread of misinformation.

Reviewer: ```suggestion
- **Public awareness and education:** Public awareness campaigns and educational initiatives will be crucial in promoting responsible use of AI-edited images and combating the spread of misinformation.
```

> This is a rapidly evolving field, and it is crucial to stay informed about the latest developments.

Reviewer 4: Do you have any sources for the ethical part people can follow - websites, social media accounts? Maybe would be nice if you know any. But if not, that is okay as well :)

Author: That sounds good, will try to add some.
